cypress/integration/core-opensearch-dashboards/opensearch-dashboards/workspace-plugin/mds_workspace_collaborators.spec.js

```diff
@@ -0,0 +1,208 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { MiscUtils } from '@opensearch-dashboards-test/opensearch-dashboards-test-library';
+const miscUtils = new MiscUtils(cy);
+const workspaceName = 'test_workspace_collaborators';
+let workspaceId;
+
+if (Cypress.env('WORKSPACE_ENABLED')) {
+  describe('Workspace collaborators', () => {
+    before(() => {
+      cy.deleteWorkspaceByName(workspaceName);
+      cy.createWorkspace({
+        name: workspaceName,
+        features: ['use-case-observability'],
+        settings: {
+          permissions: {
+            library_write: { users: ['%me%'] },
+            write: { users: ['%me%'] },
+          },
+        },
+      }).then((value) => (workspaceId = value));
+    });
+
+    beforeEach(() => {
+      // Visit workspace collaborators page
+      miscUtils.visitPage(`w/${workspaceId}/app/workspace_collaborators`);
+    });
+
+    after(() => {
+      cy.deleteWorkspaceById(workspaceId);
+    });
+
+    it('should successfully load the page', () => {
+      if (Cypress.env('SAVED_OBJECTS_PERMISSION_ENABLED')) {
```

Comment:
Can we move this check to line 11? I think the whole test file can be ignored when saved objects permissions not enabled.